tutorial06_texture/app/build.gradle

```diff
@@ -13,7 +13,9 @@ android {
 
         ndk.abiFilters 'armeabi-v7a', 'arm64-v8a', 'x86', 'x86_64'
         externalNativeBuild {
-            cmake.arguments '-DANDROID_TOOLCHAIN=clang', '-DANDROID_STL=c++_static'
+            cmake.arguments '-DANDROID_TOOLCHAIN=clang',
```

Comment:
Does this actually work with single quotes? I remember not using double quotes (") being an issue, but I may be mistaken.

Reply:
works fine for me. double quota is for macro expansion.  Single quota is "take as is"